README.md

```diff
@@ -155,22 +154,16 @@ $ yarn start
 On the **Trending News Panel**, click the RSS Feed button to launch a new tab in your browser. Cut and paste the tab URL into your favorite RSS Reader. If your RSS Feed Reader supports push notifications you can get alerted when trending topics change along with a news article for that topic.
 
 <p align="center">
-  <img width="400" src="doc/source/images/rss-2.png">
+  <img src="doc/source/images/rss-2.png">
```

Comment:
We've been moving away from HTML formatting in favor of pure markdown.

Reply:
these are long and narrow so I am keeping the <p align="center"> until i get to this step and redo the screenshots.